### PR TITLE
Propagate operator loglevel

### LINKED ISSUE #2224: Inherit loglevel setting from the operator in the sidecars
Scylla Operator creates multiple instance of its own binary, like the sidecars where we need to also adjust the loglevel. Without going into fine tuning this, inheriting the loglevel from the operator is gonna let us easily adjust it without bloating the API surface.

### PR DESCRIPTION
**Pull request by tnozicka**
**Description of your changes:**
This PR propagates operator loglevel into the sidecars to allow adjusting it there as well.

**Which issue is resolved by this Pull Request:**
Resolves #2224

### CODE REVIEW

**Comment by tnozicka** [comment]
https://github.com/scylladb/scylla-operator/issues/2118#issuecomment-2514578169
/retest

File: pkg/cmdutil/helpers.go [hunk 1]
```diff
@@ -74,11 +77,25 @@ func InstallKlog(cmd *cobra.Command) {
...
-	if f != nil {
-		return f.Value.String()
+	if f == nil {
+		return 0, errors.New(`can't lookup klog "v" flag`)
 	}
 
-	return ""
+	s := f.Value.String()
+	if len(s) == 0 {
+		return 0, nil
```
Comment (rzetelskik) [inline pkg/cmdutil/helpers.go:88]: nit: shouldn't a default have *some* verbosity?

**Review by rzetelskik — APPROVED** [review]
one nit, lgtm
/assign zimnx

File: pkg/cmdutil/helpers.go (hunk 1 above)
Comment (tnozicka) [reply pkg/cmdutil/helpers.go:88]: the default was always 0, this doesn't change it. we only usually set it to 2 explicitly in the deployments

**Review by zimnx — APPROVED** [review]
/lgtm

**Comment by tnozicka** [comment]
https://github.com/scylladb/scylla-operator/issues/2061#issuecomment-2523127350
/retest

**Review by zimnx — APPROVED** [review]
/lgtm

**Pull request merged**